### PR TITLE
Refactor cli client methods

### LINKED ISSUE #83: Client side conversion tools required
The reference server currently converts VCF data into GA4GH variant objects. It would also be useful to convert in the other direction for many purposes (for example, providing a stream to programs that do not   support GA4GH input). A useful task for the reference implementation might be to provide programs to convert GA4GH data from a given server URL and `dataSetId` to their VCF and SAM equivalents. Is this an appropriate task for the reference implementation, or should this be left to independent projects?

### PR DESCRIPTION
**Pull request by dcolligan**
Refactor in preparation for reuse of a lot of the client cli functionality by the conversion tools

Does not close #83

### CODE REVIEW

**Comment by pashields** [comment]
:star2: :+1: +1 :+1:  :star2: 

This is much easier to follow.

File: ga4gh/cli.py [hunk 1]
```diff
@@ -416,8 +417,10 @@ def client_main(parser=None):
     vsParser.set_defaults(runner=VariantSearchRunner)
     addUrlArgument(vsParser)
     addVariantSearchOptions(vsParser)
+    return vsParser
+
 
-    # variantsets/search
+def addVariantSetsSearchParser(subparsers):
     vssParser = subparsers.add_parser(
```
Comment (jeromekelleher) [inline ga4gh/cli.py]: I think we should use `parser` here as the variable name. There's no reason to come up with different variable names now, and they don't really help readability.

**Comment by jeromekelleher** [comment]
This is great, much more readable. I'm +1 after the variable name change.

**Comment by Naburimannu** [comment]
+0

**Comment by dcolligan** [comment]
Okay, just pushed a rebased change with @jeromekelleher 's suggestions

**Pull request merged**